e2e/testcafe-devextreme/tests/scheduler/grouping/overflow.ts

```diff
@@ -0,0 +1,80 @@
+import { createScreenshotsComparer } from 'devextreme-screenshot-comparer';
+import Scheduler from 'devextreme-testcafe-models/scheduler';
+import { changeTheme } from '../../../helpers/changeTheme';
+import { createWidget } from '../../../helpers/createWidget';
+import url from '../../../helpers/getPageUrl';
+
+fixture.disablePageReloads`Scheduler: SmoothCellLines`
+  .page(url(__dirname, '../../container.html'));
+
+test('Long appointments should not overflow group view', async (t) => {
+  const scheduler = new Scheduler('#container');
+  const { takeScreenshot, compareResults } = createScreenshotsComparer(t);
+
+  await t
+    .expect(await takeScreenshot('group-overflow', scheduler.element))
+    .ok()
+    .expect(compareResults.isValid())
+    .ok(compareResults.errorMessages());
+}).before(async () => {
+  await changeTheme('material.blue.light');
+
+  return createWidget('dxScheduler', {
+    timeZone: 'America/Los_Angeles',
+    dataSource: [
+      {
+        text: '1',
+        priorityId: 1,
+        startDate: new Date('2021-04-19T16:30:00.000Z'),
+        endDate: new Date('2021-04-25T18:30:00.000Z'),
```

Comment:
Minor comment

It isn't necessary to wrap the Date string into ```new Date```
```suggestion
        startDate: '2021-04-19T16:30:00',
        endDate: '2021-04-25T18:30:00',
```